orga/app/components/charts/activity.js

```diff
@@ -0,0 +1,28 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import sumBy from 'lodash/sumBy';
+
+export default class Activity extends Component {
+  @service store;
+
+  @tracked rawData = [];
+  @tracked total = 0;
+  @tracked shared = 0;
+  @tracked participantsByStatusLoading = true;
+
+  constructor(...args) {
+    super(...args);
+    const { campaignId } = this.args;
+
+    const adapter = this.store.adapterFor('campaign-stats');
+
+    adapter.getParticipationsByStatus(campaignId).then((response) => {
+      const data = response.data.attributes;
+      this.shared = data.shared;
+      this.rawData = Object.entries(data);
```

Comment:
**suggestion: (non-blocking) **: `rawData` c'est pas claire sur ce que c'est une fois que c'est passé au composant `participants-by-status`. On pourrait l'appeler `participantCountByStatus`?
Et on pourrait avoir une structure de données un peu plus pratique : `[{ status: 'shared', count: 1 }, ....]`=> `sumBy(this.rawData, 'count')`